lib/tasks/recipe_puppy_tasks.rake

```diff
@@ -0,0 +1,21 @@
+require File.expand_path('../../../config/environment', __FILE__)
+require 'utility/recipe_puppy_parser'
+
+namespace :recipe_parser do
+  desc "Scrape all recipes and ingredients from recipe puppy"
+  task :scrape_all do
+    Utility::RecipePuppyParser.new.save_all_recipes_to_database
+  end
+
+  desc "Query recipe from recipe puppy"
```

Comment:
Though I like the thinking here, it's probably unlikely that we will need to query a single recipe from the command line. We really only want the task that does the prepping of our database to be done via command line. This is so that, when we deploy our code to heroku, we can remotely trigger these tasks and populate our heroku DB from the command line.